NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -17,7 +17,7 @@
   <PropertyGroup>
     <AssemblyName>Microsoft.ApplicationInsights.AspNetCore</AssemblyName>
     <LangVersion>7.2</LangVersion>
-    <TargetFrameworks>netstandard2.0;net451;net46;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net451;net46;net461;netstandard1.6</TargetFrameworks>
```

Comment:
is there some special logic got `net451`? I cannot find a single place where `net451` or `net461 condition are used by itself. So it will be more efficient only keep `net451`.

Reply:
To enable ILogger to AspNetCore using .NETFramework, we require .NET Framework 4.6.1 (at least), with that, we will have to add net461 so we can do a condition based on the framework just what we did in standard2.0